cmd/root.go

```diff
@@ -62,6 +62,7 @@ const (
 	flagTLACodeFile = "tla-code-file"
 	flagResolver    = "resolve-images"
 	flagResolvFail  = "resolve-images-error"
+	flagExtVarEnv   = "ext-str-env"
```

Comment:
Very minor thing, but for organisation I suppose we should move this up, alongside the other `ext-str` flags

https://github.com/kubecfg/kubecfg/blob/d40858b0f72ba725c422d8cc82d6251f336d1538/cmd/root.go#L55-L58